src/solidity-helpers.ts

```diff
@@ -20,16 +20,14 @@ export interface IEthereumSigner {
  * This signer should be used for interactive signing in the browser with MetaMask.
  */
 export class Web3Signer implements IEthereumSigner {
```

Comment:
Can you create a new `EthersSigner` and leave the existing `Web3Signer` and `OfflineWeb3Signer` in place for now and just add a comment to them saying they're deprecated. We can remove them when we've gotten rid of the Web3 dependency entirely, until then they're still used in some other projects.

Reply:
Yes